flowmachine/tests/test_redacted_flows.py

```diff
@@ -17,3 +18,35 @@ def test_all_above_threshold(get_dataframe):
         ).value
         > 15
     )
+
+
+def test_in_above_threshold(get_dataframe):
+    """
+    Test that inflows redacts correctly
+    """
+    assert all(
+        get_dataframe(
+            RedactedInOutFlow(
+                in_out_flows=Flows(
+                    daily_location("2016-01-01"), daily_location("2016-01-01")
+                ).inflow()
+            )
+        ).value
+        > 15
+    )
+
+
+def test_out_above_threshold(get_dataframe):
+    """
+    Test that outflows redacts correctly
+    """
+    assert all(
+        get_dataframe(
+            RedactedInOutFlow(
+                in_out_flows=Flows(
+                    daily_location("2016-01-01"), daily_location("2016-01-01")
+                ).outflow()
+            )
+        ).value
+        > 15
+    )
```

Comment:
Might be nice to have a test that checks the summation happens before redaction (as opposed to redacting the flows first and then summing the redacted values).